io.catenax.reuse_certificate/RELEASE_NOTES.md

```diff
@@ -3,6 +3,16 @@ All notable changes to this model will be documented in this file.
 
 ## [Unreleased]
 
+## [2.0.0] - 2023-11-20
```

Comment:
please change the date to today's date if you want MS3 today.

Reply:
Sorry I just saw the message, should I still change o 11.27, or to 12.04 which is next Monday?

Reply:
You can change it to 12.04 now for our next weekly